### PR TITLE
ical: add page

### PR DESCRIPTION
**Pull request by sumanchapai**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/ical.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# ical
+
+> Program to display english Calendar on your terminal
```
Comment (marchersimon) [inline pages/common/ical.md:3]:
```suggestion
> Program to display english Calendar on your terminal.
```
According to <https://manned.org/ical>, `ical` converts the given Gregorian date to a Hirji calendar.

File: pages/common/ical.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- Display current month's calendar
```
Comment (marchersimon) [inline pages/common/ical.md:5]:
```suggestion
- Display the current month's calendar:
```

File: pages/common/ical.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+`ical`
+
+- Display the calendar of a a particular year
```
Comment (marchersimon) [inline pages/common/ical.md:9]:
```suggestion
- Display the calendar of a specific year:
```

File: pages/common/ical.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+`ical {{year eg: 1999}}`
+
+- Display the calendar for a particular  month of particular yar
```
Comment (marchersimon) [inline pages/common/ical.md:13]:
```suggestion
- Display the calendar for a specific month of a specific year:
```

File: pages/common/ical.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# ical
+
+> Program to display english Calendar on your terminal
+
```
Comment (marchersimon) [inline pages/common/ical.md:4]:
```suggestion
More information: <https://manned.org/ical>.

```
We try to put links to man pages or home pages into every page. If you find any official link for `ical` feel free to replace my suggested link.

**Review by marchersimon — CHANGES_REQUESTED** [review]
Thanks for the page @sumanchapai. I left some (mostly) syntax related suggestions below.

**Review by Waples — APPROVED** [review]

File: pages/common/ical.md (hunk 1 above)
Comment (bl-ue) [reply pages/common/ical.md]: >and vice-versa

File: pages/common/ical.md (hunk 5 above)
Comment (bl-ue) [reply pages/common/ical.md:5]: @marchersimon you frequently by mistake omit the `>` at the beginning ;)

File: pages/common/ical.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+# ical
+
+> Program to display english Calendar on your terminal.
+More information: <https://manned.org/ical>.
```
Comment (bl-ue) [inline pages/common/ical.md:4]:
```suggestion
> More information: <https://manned.org/ical>.
```

File: pages/common/ical.md (hunk 5 above)
Comment (marchersimon) [reply pages/common/ical.md:5]: Oh, thanks for reminding me

**Comment by Waples** [comment]
@sumanchapai What happend?

**Comment by sumanchapai** [comment]
I am so sorry. I was thinking that I was making PR for cal, but it looks
like I did it for ical. And upon checking again, I noticed that cal is
already done… Sorry again

**Comment by marchersimon** [comment]
Nothing to be sorry about @sumanchapai. We still appreciate the work you put in writing the page.

Now, since you already wrote the page for `ical` and it looks good overall, you could contribute that one as well if you want to.

**Comment by sumanchapai** [comment]
But I am not sure if that’s the correct manual for ical. I haven’t used it
much. So I am afraid we will be pushing incorrect things. So let’s just
hope someone will contribute the ical page. Or that I will when I am sure.

**Comment by marchersimon** [comment]
I'll have a look at it later 👍🏽

File: pages/common/ical.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
+# ical
+
+> Program to display english Calendar on your terminal.
```
Comment (marchersimon) [inline pages/common/ical.md:3]:
```suggestion
> A Hirji/Islamic calendar and converter for the terminal.
```

File: pages/common/ical.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+More information: <https://manned.org/ical>.
+
+- Display the current month's calendar:
+
+`ical`
+
+- Display the calendar of a specific year:
```
Comment (marchersimon) [inline pages/common/ical.md:10]:
```suggestion
- Convert a Gregorian date to a Hijri date:
```

File: pages/common/ical.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+
+`ical {{year eg: 1999}}`
```
Comment (marchersimon) [inline pages/common/ical.md:12]:
```suggestion
`ical --gregorian {{yyyymmdd}}`
```

File: pages/common/ical.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+
+- Display the calendar for a specific month of a specific year:
```
Comment (marchersimon) [inline pages/common/ical.md:14]:
```suggestion
- Convert a Hirji date to a Gregorian date:
```

File: pages/common/ical.md [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+
+`ical {{month, eg: 10}} {{year, eg: 1999}}`
```
Comment (marchersimon) [inline pages/common/ical.md:16]:
```suggestion
`ical --hijri {{yyyymmdd}}`
```

**Review by marchersimon — COMMENTED** [review]
With those changes and the one from @bl-ue above, the page looks alright.

**Comment by sumanchapai** [comment]
Let me commit them and open the PR in a while then.

**Review by marchersimon — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Hey there, @sumanchapai! Welcome to tldr-pages.

This page looks fine to me - thanks! A great first addition :slightly_smiling_face:

**Pull request merged**